Sources/TuistGenerator/Linter/GraphLinter.swift

```diff
@@ -170,6 +172,26 @@ public class GraphLinter: GraphLinting {
         }
     }
 
+    private func lintAppClip(graph: Graph) -> [LintingIssue] {
```

Comment:
nice work 👍 

Seeing we're only ever extracting one `.appClip` product in the graph logic, it's worth us adding another lint rule if we detect an app has more than one app clip.

Reply:
Good catch, the lint rule is definitely needed, thanks